srcpkgs/python3-jinja2-time/patches/arrow-replace-to-shift.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/jinja2_time/jinja2_time.py b/jinja2_time/jinja2_time.py
+index ce713cb..717c8a0 100755
+--- a/jinja2_time/jinja2_time.py
++++ b/jinja2_time/jinja2_time.py
+@@ -19,11 +19,11 @@ class TimeExtension(Extension):
+         d = arrow.now(timezone)
+ 
+         # Parse replace kwargs from offset and include operator
+-        replace_params = {}
++        shift_params = {}
+         for param in offset.split(','):
+             interval, value = param.split('=')
+-            replace_params[interval.strip()] = float(operator + value.strip())
+-        d = d.replace(**replace_params)
++            shift_params[interval.strip()] = float(operator + value.strip())
++        d = d.shift(**shift_params)
+ 
+         if datetime_format is None:
+             datetime_format = self.environment.datetime_format
```

Comment:
Why does this patch exist?

Reply:
jinja2-time hasn't received an update for a long time, this is purely to let it interact with new versions of python3-Arrow

Reply:
The only two real commits in the last six years have been to switch CI providers an limit dependency versions on old Python. This looks like abandonware.

Reply:
maybe, but it's still a dependency :/